tutorials/javaFxPart1.md

```diff
@@ -109,6 +109,30 @@ public class Duke extends Application {
 }
 ```
 
+However, if you do already have an existing constructor which takes arguments, you can proceed by creating an overloaded constructor with no arguments, passing the arguments to the existing constructor e.g.,
+
+```java{highlight-lines="11-14"}
+//...
+private static final String DEFAULT_FILE_PATH = "duke/example.txt";
+
+public class Duke extends Application {
+
+    // Existing constructor
+    public Duke(String filePath) {
+        // ...
+    }
+
+    // Overloaded constructor
+    public Duke() {
+        this(DEFAULT_FILE_PATH);
+    }
+
+    // ...
+}
+```
+
+This approach enables JavaFX to create the `Application` instance using the newly defined 
+no-argument constructor, while preserving the functionality of the existing constructor.
```

Comment:
This sounds like we are doing an optional thing, but IIRC, this is a requirement based on how JavaFX works, right?

Also, unnecessary line break.

Reply:
According to the [documentation](https://openjfx.io/javadoc/21/javafx.graphics/javafx/application/Application.html), yes, we do need to create a no-argument constructor in order to call `Application::launch()`.

Reply:
I was thinking of adding an elaboration as to why the user would need to do so, but it seems to already be mentioned in line 97 of `javaFxPart1.md`.

```markdown
JavaFX creates the `Application` instance by calling the no-argument constructor. So, you need to define a no-argument constructor in the `Duke` class (i.e., the `Application` subclass you use) if you don't have one already e.g.,
```